.github/actions/height-integration/sync-pr/pr-opened-action.yml

```diff
@@ -0,0 +1,18 @@
+name: Sync new PR to Height
+description: Moves any issue-related tasks in Height into the 'In Progress' swimlane on the interface project board
```

Comment:
What happens with this one if its from community?

Reply:
My suspect is it doesn't matter what the source is coming from, as the same `pull_request['opened','reopened']` still applies.  I suspect the event will trigger and our action will run and PUT to Height API.  I think this action is making an assumption that _could be there_ but most likely _won't be there.  And that is that _the issue would already be linked somehow to this brand new PR.  This is part of the steps in our checklist, to add the issue in Development. If the author adds this in while creating the PR the first time, it'll get picked up and I believe we'd easily have access to knowing what issues it's related to during this PUT request.  However, not everyone remembers/does that (myself included) to I believe the assumption is okay, but not 100%.  This is team workflow/behavior curbing that we could do and fix in theory.  And also, another behavior is that we create issues for _everything_ we're setting up PRs with, so that they get into Height regardless and are tracked, chores and all.  With the automation, that makes our lives easy, but its the behavior/workflow we just start curbing and practicing.

Now, for original question though, we _could_ write in this same action listening to the `workflow_approve` and/or `workflow_run` events, as we'd need to do that for forks anyway to get an environment built if it's a valid request/feature.  Check out all the events here https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
After typing this out and thinking it through, to solve our woes for development linking on PR creation, we could listen to if that event happens with the `pull_request['edited']` I think